_posts/2022-09-06-Umesh-GSoC-on-ConscryptMode.md

```diff
@@ -0,0 +1,24 @@
+---
+title:  "GSoC 2022 final submission for ConscryptMode"
+author: Umesh-01
+---
+
+My name is Umesh Singh and I was an open source contributor through Google Summer of Code this year. Google Summer of Code (GSoC) is a program where external participants can contribute to an open source project over a few months. We learn new computer science concepts, how to work on an open source repository, and create real contributions to Google code!
```

Comment:
`We learn new computer science concepts, how to work on an open source repository, and create real contributions to Google code!` -> `We learn new computer science concepts, how to work on open source repositories, and create real code contributions to projects!`

Robolectric is not Google code, it's open-source :)

Reply:
done